src/components/Layout/Layout.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
I have an idea.
**What if we have 3 types of Layouts**
1. AccountLayout.js where we will render admin pages such as profile, notifications, setting, and other pages that appear in the account of a logged in user. The difference with other pages is that these ones have the account navigation bar and its own way of grids. 
2. PublicLayout.js where we will render public pages such as read article, Home pages, ....(shouldn't require authentication). The difference with other pages is that these ones do not have the account navigation bar
3. AuthLayout.js where we will render pages like login, register, reset the password, request the password, ...  The difference with other pages is that these ones do not have the header, the account navigation bar, the grids are different too, etc.

They have a different view, hence  it can be good if we render them separately 
What do you think guys? @rwajon @sengayire @luc-tuyishime @MBenedicte

Reply:
I think this can be an approach that we can consider

Reply:
what's the consensus on this?

Reply:
@e-liyai  For now, we decided to use one layout as we are yet to have a final idea on how many layouts we need